packages/api/src/external/carequality/command/cq-directory/cq-gateways.ts

```diff
@@ -14,13 +14,23 @@ export async function setEntriesAsGateway(oids: string[]): Promise<void> {
   );
 }
 
-export async function getOrganizationIds(excludeManagingOrgs: string[]): Promise<string[]> {
+/**
+ * Returns the ID of CQ directory entries that are not managed by the organizations provided as
+ * parameter.
+ *
+ * @param managingOrgNames - An array of managing organization names
+ * @returns IDs of the CQ directory entries not managed by the provided orgs
+ */
+export async function getOrganizationIdsNotManagedBy(
+  managingOrgNames: string[]
+): Promise<string[]> {
   const entries = await CQDirectoryEntryModel.findAll({
     attributes: ["id"],
     where: {
-      managingOrganization: {
-        [Op.notIn]: excludeManagingOrgs,
-      },
+      [Op.or]: [
+        { managingOrganization: { [Op.is]: undefined } },
```

Comment:
is this new condition necessary? like is their an edge case this is catching?

Reply:
Sequelize's `notIn` doesn't include `null` values, so this accounts for that.

Reply:
great sounds good